src/Form/Fields.php

```diff
@@ -184,6 +184,20 @@ public function findByKeyRecursive(string $key): Field|FieldClass|null
 		return $field;
 	}
 
+	/**
+	 * Creates a new Fields instance for the given model and language
+	 */
+	public static function for(
+		ModelWithContent $model,
+		Language|string $language = 'default'
```

Comment:
`Question:` is `current` maybe the more common use case?

Reply:
A bit torn here. We mix this. In the version classes, we use default and in form and content method, we often use current.